OSDOCS#13054-2: Adds missing extension type to the dynamic plugin reference docs (#88586)
openshift/openshift-docs: modules/dynamic-plugin-sdk-extensions.adoc

|`useValueHook` |`CodeRef<() => T>` |no |Hook for the Context value.
|===

[discrete]
== `console.create-project-modal`

> This extension can be used to pass a component that will be rendered in place of the standard create project modal.

Reviewer: ```suggestion
You can use this extension to pass a component that is rendered in place of the standard create project modal.
```

In the preview, on this page I see other passive voice instances too. So you might want to keep the consistency. Also I am wondering if this plugin reference doc is generated automatically like the Rest API. (!)

Author: Hey! Not Generated automatically like Rest API, but these are like the other API docs. I copy them directly from the console.

Author: We would treat these like the other API docs.

Reviewer: Ah ok. Good to know :)
